src/main/java/dev/thedocruby/resounding/mixin/WorldChunkMixin.java

```diff
@@ -143,21 +143,22 @@ public void initStorage() {
 		// 16³ blocks
 		ChunkSection[] chunkSections = getSectionArray();
 		// LOGGER.info(String.valueOf(chunkSections.length) /* +"\t"+Arrays.toString(chunkSections)*/); // TODO remove
-		this.yOffset = -chunkSections[0].getYOffset() >> 4;
+		this.yOffset = -heightLimitView.sectionIndexToCoord(0);
```

Comment:
Could you explain this workaround?

Reply:
This workaround involved looking at how the value returned by `ChunkSection#getYOffset()` was calculated in `1.18.2`. I had to see how this value was calculated and then redid as little of it with what is available in `1.21`. I am unsure if the descriptions of commits bbc38419d3ff4ff3c82f34870773d03cb9c94e6d and 66dba687f17957e22afdf66b442974a56f09d281 will suffice as this may need verification by looking at the decompiled code for Minecraft `1.18.2`.